src/js/components/scroll-indicator.js

```diff
@@ -0,0 +1,26 @@
+import { newText, newContainer } from './helpers';
+
+const scrollIndicator = (data) => {
+  const button = document.createElement('button');
+  button.type = 'button';
+  const image = document.createElement('img');
```

Comment:
Please use Helpers function to create each component. This will help standardize the code
This will help standardize the code

Reply:
all done Cami, thanks